dockerfiles/Dockerfile.ubuntu

```diff
@@ -67,13 +67,13 @@ RUN echo "ServerName localhost" >> /etc/apache2/apache2.conf
 WORKDIR /var/www/html
 RUN rm index.html
 
-COPY . /var/www/html
-RUN curl -sS https://getcomposer.org/installer | php
-RUN php composer.phar install --no-dev
+# COPY . /var/www/html
+# RUN curl -sS https://getcomposer.org/installer | php
+# RUN php composer.phar install --no-dev
 
 
 # Configure Skosmos
-COPY dockerfiles/config/config-docker.ttl /var/www/html/config.ttl
+# COPY dockerfiles/config/config-docker.ttl /var/www/html/config.ttl
```

Comment:
Note to self: this is to speed up development, mapping the Docker volume. Undo all the changes to the Dockerfiles when the PR is ready for review.